src/components/InstalledPage.jsx

```diff
@@ -20,13 +21,25 @@ const DumbInstalledPage = props => {
           <img src={extensionInstalledIcon} alt="" />
           <Stack spacing="m">
             <MainTitle>{t('InstalledPage.title')}</MainTitle>
-            <Text
-              dangerouslySetInnerHTML={{
-                __html: snarkdown(
-                  t('InstalledPage.description', { address: cozyURL.host })
-                )
-              }}
-            />
+            <Text>
+              <span
+                dangerouslySetInnerHTML={{
+                  __html: snarkdown(
+                    t('InstalledPage.descriptionStart', {
+                      address: cozyURL.host
+                    })
+                  )
+                }}
+              />{' '}
+              <WithCozyIcon>cozy</WithCozyIcon>{' '}
```

Comment:
Nit: I am not sure using a super specific component like WithCozyIcon is needed here. Maybe a Media could suffice ? `<Media><Img><CozyIcon/></Img><Bd>cozy</Bd></Media>` A bit more characters to type but we avoid building super specific components.

Reply:
Implementing it using Media, Img and Bd is great because we can drop the custom CSS. But since we have two places at which we have this kind of text with a cozy icon, and the tree to get the good visual rendering is:

```jsx
    <Media className="u-inline-flex">
      <Img>
        <Icon icon="cloud" size={16} color="var(--charcoalGrey)" />
      </Img>
      <Bd className="u-ml-half">
        <Bold tag="strong" {...props} />
      </Bd>
    </Media>
```

I think we should keep a specialized component.

Reply:
See https://github.com/cozy/cozy-passwords/pull/1/commits/520165ef9b76fc8dcf2387610303f41a2f1928c7